app/com/yetu/oauth2provider/oauth2/models/YetuUser.scala

```diff
@@ -23,15 +24,16 @@ case class YetuUser(identityId: IdentityId, uid: String, firstName: String, last
     registrationDate: Option[Date] = None,
     contactInfo: Option[ContactInfo] = None,
     imageUrl: Option[String] = None,
-    publicKey: Option[YetuPublicKey] = None) {
+    publicKey: Option[YetuPublicKey] = None,
+    userAgreement: Option[UserAgreement] = None) {
 
   //TODO: clean this up.
```

Comment:
At the moment, securesocial expects a "BasicProfile" when doing login/registration. But there is also a YetuUser, as we wish to store a few extra fields, like e.g. the public SSH key. So then there are conversions between YetuUser and BasicProfile back and forth, and with this commit we also changed the BasicProfile itself. So if we do that anyway, we do not need to maintain two different kinds of "User" Types that are mostly the same but slightly different for historic users. That should be cleaned up, either by removing one case class, or by extracting additional fields into a separate class.